microsoft365defender_connector.py

```diff
@@ -1150,6 +1150,164 @@ def _handle_update_alert(self, param):
 
         return action_result.set_status(phantom.APP_SUCCESS, DEFENDER_ALERT_UPDATED_SUCCESSFULLY)
 
+    def _check_invalid_since_utc_time(self, action_result, time):
+        """Determine that given time is not before 1970-01-01T00:00:00Z.
+        Parameters:
+            :param action_result: object of ActionResult class
+            :param time: object of time
+        Returns:
+            :return: status(phantom.APP_SUCCESS/phantom.APP_ERROR)
+        """
+        # Check that given time must not be before 1970-01-01T00:00:00Z.
+        if time < datetime.strptime("1970-01-01T00:00:00Z", DEFENDER_APP_DT_STR_FORMAT):
+            return phantom.APP_ERROR
+        return phantom.APP_SUCCESS
```

Comment:
```suggestion
        return time >= datetime.strptime("1970-01-01T00:00:00Z", DEFENDER_APP_DT_STR_FORMAT)
```
Why not use a regular bool but a wrapper?

Reply:
Done